app/src/main/java/com/example/palletify/ui/generator/GeneratorViewModel.kt

```diff
@@ -111,7 +111,6 @@ class GeneratorViewModel : ViewModel() {
             currentState.copy(
                 numberOfColours = palette.numberOfColours,
                 currentPalette = palette.colors.toMutableList(),
-                mode = palette.mode,
```

Comment:
Removed this so that the dropdown menu is the "source of truth" that has control over ui state mode. This way, when the mode is ANY and a palette is generated with a random mode (i.e. MONOCHROME), the palette mode is MONOCHROME but the uistate's mode remains ANY.